lib/TeleIrc.js

```diff
@@ -442,11 +449,22 @@ class TeleIrc {
    * Sends a message to the IRC channel.
    * @param {String} message - The message to send to the IRC channel.
    */
-  sendMessageToIrc(message) {
-    this.ircbot.say(
-      this.config.irc.channel,
-      message
-    );
+  sendMessageToIrc(from, message) {
+    const ircbot = this.ircbot;
+    const messagePrefix = this.config.irc.prefix + from + this.config.irc.suffix + " ";
+    const channel = this.config.irc.channel;
+    const maxLength = this.config.irc.maxMessageLength;
+
+    message.toString().split(/\r?\n/).filter(function(line) {
```

Comment:
Looks like the `toString()` method is failing when the `message` is an image, which naturally wouldn't have a `toString` method.